### ISSUE TITLE
[Slider] Lighthouse accessibility warnings

### ISSUE THREAD

**Issue by david-roeger** [radix-ui/primitives#1339]
## Bug report

Running Lighthouse on Slider Component raises accessibility issues. "ARIA input fields do not have accessible names" for the thumb element. Running AXE also confirms this issue with an additional ssue "Elements must only use allowed ARIA attributes" (aria-label attribute is not well supported on a span with no valid role attribute) for the Root Element.

### Current Behavior
<img width="624" alt="image" src="https://user-images.githubusercontent.com/61949735/164678587-f4d2aac8-43f5-4a5f-b7f1-be848826e18e.png">

### Expected behavior
I´m not quite sure if this is the expected behavior  and the current aria-patterns are valid. Maybe someone can confirm. But i would expect not to raise an issue in lighthouse 

### Reproducible example
Normal sandbox from the slider docs
https://codesandbox.io/s/j4veyp?module=App.js

### Your environment

| Software         | Name(s) | Version |
| ---------------- | ------- | ------- |
| Radix Package(s) |     react-slider    |    latest    |
| React            | n/a     |     latest    |
| Browser          |      Chrome   |   Version 100.0.4896.127       |
| Assistive tech   |    -     |     -    |
| Node             | n/a     |     -    |
| npm/yarn         |     -    |     -   |
| Operating System |    macOS Monterey    |    12.0.1 (21A559)     |

**Comment by benoitgrelard** [comment]
Hey @david-roeger,

I am not sure how relevant that is as in practice screen readers will announce the slider fine with its `aria-label`…

**Comment by david-roeger** [comment]
@benoitgrelard 
Thanks for the quick answer.
Besides from my light house score going down there are no negative side effects. Everything else works as expected 👍

**Comment by benoitgrelard** [comment]
Yeah it's not the first time we see false negatives like that with lighthouse.

**Comment by gunningham** [comment]
Similar issue for me, the Label component is a span with role="label", which isn't a valid role. What's the harm in simply using a label element?

**Comment by benoitgrelard** [comment]
Hi @gunningham, what you are mentioning has already been discussed in #638 where we've explained original thoughts.
That said, we are also investigating if we can now rely on native `label` in #1233.

**Comment by benoitgrelard** [comment]
We're going to close this down as this isn't a real accessibility problem but rather a reporting one with those tools.

**Comment by vincerubinetti** [comment]
Ran into this "problem" with the Vue port of Radix. Just to be clear, `aria-label` isn't intended to be there, and SRs should be able to find the parent label and announce it that way? That makes sense to me, and I'm surprised Lighthouse, being the golden standard, throws a false positive.
